app/query_param.go

```diff
@@ -48,3 +48,29 @@ func ParseHistoriesQueryParams(c echo.Context, defaultLimit int64) ([]string, st
 
 	return topicsSuffix, userID, from, limit
 }
+
+func ParseHistoryPSQueryParams(c echo.Context, defaultLimit int64) (string, string, string, int64, int64, int64, bool) {
+	userID := c.QueryParam("userid")
+	playerId := c.QueryParam("playerId")
+	topic := c.QueryParam("topic")
+	from, _ := strconv.ParseInt(c.QueryParam("from"), 10, 64)
```

Comment:
Does it make sense to retrieve `from` and `to` if they aren't used on the new endpoint directly? (the endpoint is parsing them in separate after this call)

Reply:
> 

Yes, Dani! It does!